cms/static/sass/views/_certificates.scss

```diff
@@ -0,0 +1,723 @@
+// studio - views - certificates
+// ====================
+.view-certificates {
+
+  .content-primary, .content-supplementary {
+    @include box-sizing(border-box);
+    float: left;
+  }
+
+  .content-primary {
+    width: flex-grid(9, 12);
+    margin-right: flex-gutter();
+
+    .notice-moduledisabled {
+      @extend %ui-well;
+      @extend %t-copy-base;
+      background-color: $white;
+      padding: ($baseline*1.5) $baseline;
+      text-align: center;
+    }
+
+    .no-content {
+      @extend %no-content;
+    }
+
+    .wrapper-certificate {
+      margin-bottom: ($baseline*1.5);
+
+      .title {
+        @extend %t-title4;
+        @extend %t-strong;
+        margin-bottom: ($baseline/2);
+      }
+
+      .copy {
+        @extend %t-copy-sub1;
+      }
+    }
+
+    .wrapper-collection {
+      @extend %ui-window;
+      position: relative;
+      outline: none;
+
+      &:hover .collection .actions {
+        opacity: 1.0;
+      }
+
+      .collection-details {
+        padding: $baseline ($baseline*1.5);
+      }
+
+      .collection-header {
+        margin-bottom: 0;
+        border-bottom: 0;
+        padding-bottom: 0;
+
+        .title {
+          @extend %cont-truncated;
+          @extend %t-title5;
+          @extend %t-strong;
+          margin-right: ($baseline*14);
+          color: $black;
+
+          .toggle {
```

Comment:
remove styles for toggling from this view (lines 58 -->93 or so) This also should be removed from HTML.